packages/core/components/DataTable/DataTable.tsx

```diff
@@ -125,6 +125,16 @@ const DataTable = (props: DataTableProps) => {
     maxHeight: config.table.limitHeight && `${config.table.height}px`,
     OverflowY: 'scroll'
   }
+  const [height, getTableHeight] = useState(0)
+  const tableHgt = useRef(null)
+  useEffect(() => {
+    getTableHeight(tableHgt.current.clientHeight)
+  }, [])
+
+  const fixedHeight = {
```

Comment:
The ref is always returning zero for me here, so the table is not displaying for me.